src/Bicep.Core.UnitTests/Assertions/RootConfigurationAssertions.cs

```diff
@@ -25,7 +26,7 @@ public AndConstraint<RootConfigurationAssertions> HaveContents(string contents,
         {
             var actual = Subject.ToUtf8Json().ReplaceLineEndings();
             var expected = contents.ReplaceLineEndings();
-            actual.Should().Be(expected, because, becauseArgs);
+            JsonAssert.AreEqual(expected, actual, output: true);
```

Comment:
We already have a method to do this - please could you switch this to using `.Should().DeepEqual()` instead?

```suggestion
            JToken.Parse(expected).Should().DeepEqual(JToken.Parse(actual));
```